modules/getting-started/con-audit-log-config.adoc

```diff
@@ -0,0 +1,10 @@
+[id="con-audit-log-config_{context}"]
+= Configuring the {product-short} audit log for {ocp-short}
+
+You can use the {ocp-short} web console to configure the following {ocp-short} logging components to use audit logging for {product-short}:
+
+*Logging deployment*: Configure the logging environment, including both the CPU and memory limits for each logging component. For more information, see link:https://docs.openshift.com/container-platform/4.15/observability/logging/config/cluster-logging-memory.html[Configuring your Logging deployment] in the {ocp-short} documentation.
+
+*Logging collector*: Configure the `spec.collection` stanza in the ClusterLogging custom resource (CR) to use a supported modification to the log collector. For more information, see link:https://docs.openshift.com/container-platform/4.15/observability/logging/log_collection_forwarding/cluster-logging-collector.html[Configuring the logging collector] in the OpenShift Container Platform documentation.
+
+*Log forwarding*: To send logs to specific endpoints inside and outside your {ocp-short} cluster, you specify a combination of outputs and pipelines in a `ClusterLogForwarder` CR. For more information, see link:https://docs.openshift.com/container-platform/4.15/observability/logging/log_collection_forwarding/cluster-logging-enabling-json-logging.html[Enabling JSON log forwarding] and link:https://docs.openshift.com/container-platform/4.15/observability/logging/log_collection_forwarding/configuring-log-forwarding.html[Configuring log forwarding] in the {ocp-short} documentation.
```

Comment:
```suggestion
You can use the {ocp-short} web console to configure the following {ocp-short} logging components to use audit logging for {product-short}:

*Logging deployment*: Configure the logging environment, including both the CPU and memory limits for each logging component. For more information, see link:https://docs.openshift.com/container-platform/4.15/observability/logging/config/cluster-logging-memory.html[Configuring your Logging deployment] in the {ocp-short} documentation.

*Logging collector*: Configure the `spec.collection` stanza in the ClusterLogging custom resource (CR) to use a supported modification to the log collector. For more information, see link:https://docs.openshift.com/container-platform/4.15/observability/logging/log_collection_forwarding/cluster-logging-collector.html[Configuring the logging collector] in the OpenShift Container Platform documentation.

*Log forwarding*: To send logs to specific endpoints inside and outside your {ocp-short} cluster, you specify a combination of outputs and pipelines in a `ClusterLogForwarder` CR. For more information, see link:https://docs.openshift.com/container-platform/4.15/observability/logging/log_collection_forwarding/cluster-logging-enabling-json-logging.html[Enabling JSON log forwarding] and link:https://docs.openshift.com/container-platform/4.15/observability/logging/log_collection_forwarding/configuring-log-forwarding.html[Configuring log forwarding] in the {ocp-short} documentation.
```

I find this section confusing. It's not clear what we're telling users to configure, or what the use cases for configuring these things would be, and pointing to the logging docs doesn't give any RHDH-specific information. What exactly do we mean here by "configure the following {ocp-short} logging components to use audit logging for {product-short}"? In what way are they using the audit logs, and to achieve what function?

I have a feeling we're maybe trying to tell users that they can configure OCP logging to:

* set limits on the amount of audit logs / memory etc, in which case we should give any info about the approx. size of RHDH audit logs, what's the minimum requirements, maybe how they can filter important vs unimportant, etc.
* collect audit logs for RHDH - IIRC, we use a http receiver for logging to get audit logs (https://docs.openshift.com/container-platform/4.15/observability/logging/log_collection_forwarding/cluster-logging-collector.html#log-collector-http-server_cluster-logging-collector) so idk if there's anything specific you need to do for RHDH for that. We also say "If you configure the log collector to collect audit logs, it collects them from /var/log/audit/audit.log" so maybe if the RHDH audit logs go there automatically it's not an issue and you just configure it on the logging end, but idk how that works for RHDH
* then forward audit logs from RHDH to a third party e.g. Elasticsearch

If that's what we want to do, we should have a specific module in our docs describing how to do that, and then provide additional links to OCP logging docs if there's more in-depth conceptual stuff we want to show. https://docs.openshift.com/container-platform/4.15/observability/logging/log_collection_forwarding/configuring-log-forwarding.html#cluster-logging-collector-log-forwarding-about_configuring-log-forwarding shows examples of how audit log forwarding is configured for logging, so maybe we want to show how to specifically configure something in the `ClusterLogForwarder` CR for RHDH? That's maybe a question for eng though since I'm not positive about the use cases here, just guessing.

Reply:
@schultzp2020 do we need to include any RHDH-specific information or examples to help the user configure these OCP logging components for RHDH?

Reply:
There shouldn't be any RHDH-specific information. As long as the examples show collecting logs from STDOUT, we're good.